core/pom.xml

```diff
@@ -208,10 +172,12 @@
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
```

Comment:
Only this scope was a regression from my previous changes. Those test dependencies were wrongly packaged in the artifact...